sandbox/libs/authn/src/test/java/org/opensearch/authn/AuthenticationTokenHandlerTests.java

```diff
@@ -17,7 +18,7 @@
 
 public class AuthenticationTokenHandlerTests extends OpenSearchTestCase {
 
-    public void testShouldExtractBasicAuthTokenSuccessfully() {
+    public void testShouldExtractBasicAuthTokenSuccessfully() throws BadCredentialsException {
```

Comment:
these changes won't be needed anymore as the signature for extractShiroAuthToken will be changed to not throw BadCredentialsException. (See comments above)